dfs/destroy.go

```diff
@@ -22,31 +22,34 @@ import (
 func (dfs *DistributedFilesystem) Destroy(tenantID string) error {
 	vol, err := dfs.disk.Get(tenantID)
 	if err != nil {
-		glog.Errorf("Could not get volume for tenant %s: %s", tenantID, err)
-		return err
-	}
-	snapshots, err := vol.Snapshots()
-	if err != nil {
-		glog.Errorf("Could not get snapshots for tenant %s: %s", tenantID, err)
-		return err
-	}
-	for _, snapshot := range snapshots {
-		if err := dfs.Delete(snapshot); err != nil {
-			glog.Errorf("Could not remove snapshot %s for tenant %s: %s", snapshot, tenantID, err)
-			return err
+		glog.Errorf("Error destroying DFS:  Could not get volume for tenant %s: %s", tenantID, err)
```

Comment:
If we return here, then the dfs.disk.Remove call below will never happen, which we need in order to remove the volume itself (or just the metadata if the volume is gone).